src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/SplitContainer.SplitContainerAccessibleObjectTests.cs

```diff
@@ -67,11 +67,55 @@ public void SplitContainerAccessibleObject_GetPropertyValue_ControlType_IsExpect
             using SplitContainer splitContainer = new SplitContainer();
             splitContainer.AccessibleRole = role;
 
-            object actual = splitContainer.AccessibilityObject.GetPropertyValue(UiaCore.UIA.ControlTypePropertyId);
-            UiaCore.UIA expected = AccessibleRoleControlTypeMap.GetControlType(role);
+            object actual = splitContainer.AccessibilityObject.GetPropertyValue(UIA.ControlTypePropertyId);
+            UIA expected = AccessibleRoleControlTypeMap.GetControlType(role);
 
             Assert.Equal(expected, actual);
             Assert.False(splitContainer.IsHandleCreated);
         }
+
+        [WinFormsTheory]
+        [InlineData((int)UIA.NamePropertyId, "TestName")]
+        [InlineData((int)UIA.AutomationIdPropertyId, "SplitContainer1")]
+        public void SplitContainerAccessibleObject_GetPropertyValue_Invoke_ReturnsExpected(int propertyID, object expected)
+        {
+            using var control = new SplitContainer
+            {
+                Name = "SplitContainer1",
+                AccessibleName = "TestName"
+            };
+
+            var accessibleObject = new SplitContainer.SplitContainerAccessibleObject(control);
```

Comment:
Please use one style for all tests, as you did before:
```suggestion
            using SplitContainer control = new()
            {
                Name = "SplitContainer1",
                AccessibleName = "TestName"
            };

            SplitContainer.SplitContainerAccessibleObject accessibleObject = new(control);
```